openmc/plotter.py

```diff
@@ -27,10 +27,37 @@
 UNITY_MT = -1
 XI_MT = -2
 
+def get_reaction_components(mt):
```

Comment:
[PEP8 requests two blank lines](https://www.python.org/dev/peps/pep-0008/#blank-lines) before and after module level functions
```suggestion

def get_reaction_components(mt):
```